lib/philomena_web/controllers/api/json/comment_controller.ex

```diff
@@ -0,0 +1,26 @@
+defmodule PhilomenaWeb.Api.Json.CommentController do
+  use PhilomenaWeb, :controller
+
+  alias PhilomenaWeb.CommentJson
+  alias Philomena.Comments.Comment
+  alias Philomena.Repo
+  import Ecto.Query
+
+  def show(conn, %{"id" => id}) do
+    comment = 
+      Comment
+      |> where(id: ^id)
+      |> preload([:image, :user])
+      |> Repo.one()
+
+    case comment do
+      nil ->
+        conn
+        |> put_status(:not_found)
+        |> text("")
+
+      _ ->
+        json(conn, %{comment: CommentJson.as_json(comment)})
```

Comment:
The case of deleted comments (any which are marked as `hidden_from_users`/`destroyed_content`) needs to be handled in `CommentJson` so as to not end up displaying their contents